README.md

```diff
@@ -856,6 +856,49 @@ be installed from the regular repository. Defaults to `true`.
 If the postgresql.org repo is installed, you can install several versions of
 postgres. Defaults to `9.4`, the latest stable version.
 
+### puppetdb::cli
+
+The `puppetdb::cli` class manages the `puppet-client-tools` package which
+contains the PuppetDB CLI. This class also manages the associated PuppetDB CLI
+configuration file.
+
+You must declare the class to use it:
+
+    class { 'puppetdb::cli': }
+
+**Parameters within `puppetdb::cli`:**
+
+####`version`
+
+The version of the `puppet-client-tools` package that should be installed. You
+may specify an explicit version number, 'present', or 'latest' (defaults to
+'present').
+
+####`puppetdb_urls`
```

Comment:
why invent a new term here? is it going to conflict with server_urls somewhere else?